services/users.js

```diff
@@ -87,6 +87,10 @@ async function getSocialCircle (secret) {
   return seen
 }
 
+async function deleteUserAndFriends (secret) {
+  return UserModel.deleteUserAndFriendships()
```

Comment:
Don't we need to pass `secret` to `.deleteUserAndFriendships()`?

Also [nit] I think we should rename to `deleteUserAccount` -- services should be high-level ideas, whereas `deleteUserAndFriendships` fits `UserModel` as a data-layer function name

Reply:
:o wait wat. I dont understand how it was working without passing secret lmao. Good catch.